chemview/viewer.py

```diff
@@ -54,6 +54,31 @@ def update(self=self, points=points):
         self.update_callbacks.append(update)
         self.autozoom(self.coordinates)
         
+    def labels(self, atom_labels=None, colors=None):
+        '''Display atomic labels for the system'''
+        if not atom_labels:
+            if len(self.topology.get('atom_types'))==len(self.coordinates):
+                atom_labels=[self.topology['atom_types'][i]+str(i+1) for i in range(len(self.coordinates))]
+            else:
+                atom_labels=[str(i+1) for i in range(len(self.coordinates))]
+        if not colors:
+            colors=[{"r":255,"g":255,"b":0,"a":1} for a in self.coordinates]
```

Comment:
To be consistent with the rest of the application the colors should be integers eg (white = 0xffffff) and alpha values should be passed as an extra argument ( each alpha value is a floating point number between 0 and 1).